libraries/botbuilder-core-extensions/lib/recognizerResult.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"recognizerResult.js","sourceRoot":"","sources":["../src/recognizerResult.ts"],"names":[],"mappings":";AAAA;;GAEG;AACH;;;EAGE"}
```

Comment:
There's no src file... If you didn't make an changes to core-extensions could you revert all of these lib files?